fe/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -162,7 +162,7 @@ protected void runPendingJob() throws AlterCancelException {
             }
         }
         MarkedCountDownLatch<Long, Long> countDownLatch = new MarkedCountDownLatch<Long, Long>(totalReplicaNum);
-        db.readLock();
+        db.writeLock();
```

Comment:
Could we reduce the write lock scope? because the `tbl.isStable` and `CreateReplicaTask` both need to iterate every replica.

Reply:
OK, I will modify this.